src/ngraph/op/convert.cpp

```diff
@@ -55,3 +56,38 @@ void op::Convert::generate_adjoints(autodiff::Adjoints& adjoints, const OutputVe
 
     adjoints.add_delta(x, make_shared<op::Convert>(delta, x.get_element_type()));
 }
+
+namespace
+{
+    template <element::Type_t ET>
+    bool try_evaluate_convert(const EvaluatorTensorPtr& arg,
+                              const EvaluatorTensorPtr& out,
+                              size_t count)
+
+    {
+        return (ET == arg->get_element_type()) &&
+               (runtime::reference::convert(arg->get_ptr<ET>(), out->get_ptr<ET>(), count), true);
+    }
+
+    bool
+        evaluate_convert(const EvaluatorTensorPtr& arg, const EvaluatorTensorPtr& out, size_t count)
+    {
```

Comment:
You will need to set the output element type for `out` here, and copy the output shape from `arg` to `out`

Reply:
done

Reply:
done